src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js

```diff
@@ -90,7 +57,6 @@ const ViewPaymentHistoryCTA = props => {
 };
 
 const mapStateToProps = store => ({
-  includedInFlipper: toggleValues(store)[FEATURE_FLAG_NAMES.viewPaymentHistory],
```

Comment:
Can we go a step further by removing this widget in favor of a [call-to-action widget](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/cta-widget/helpers.js#L11)? If you aren't familiar, the green alertbox [here](https://www.va.gov/change-direct-deposit/) is an example

Reply:
Can do!